test/test_repros.py

```diff
@@ -1490,6 +1499,16 @@ def to_bitmasks(boxes):
 
         self.assertTrue((to_bitmasks(torch.zeros(10)) == torch.ones(10)).all())
 
+    @unittest.skipIf(not has_scipy(), "requires scipy")
+    def test_scipy_import(self):
+        def fn():
+            import scipy.sparse
+
+            print(scipy.sparse.coo_matrix)
+
+        opt_fn = torchdynamo.optimize("eager")(fn)
+        opt_fn()
```

Comment:
You don't need scipy to trigger this bug, no reason to make the test case specialized for scipy